opencti-platform/opencti-front/src/private/components/observations/stix_cyber_observables/StixCyberObservableHeader.jsx

```diff
@@ -49,27 +48,24 @@ const StixCyberObservableHeaderComponent = ({
       </Typography>
       <div className={classes.actions}>
         <div className={classes.actionButtons}>
-          {disableSharing !== true && (
```

Comment:
Why no more condition?

Reply:
this option was simply never used. We never disable sharing.